analyzeDelphes.py

```diff
@@ -9,7 +9,8 @@
     21:"gamma", #using this for all jets
```

Comment:
I just noticed this is a bug.  Photons are pdgID 22, gluons are 21
```suggestion
    21:"j", #using this for all jets
    22:"gamma",
```